examples/node-app-pg/src/sql/function/dbo/sum.sql

```diff
@@ -0,0 +1,5 @@
+CREATE FUNCTION [dbo].square(@x INT) RETURNS INT
+AS
+BEGIN
+    RETURN @x * @x;
+END
```

Comment:
@Prabeshpd You are missing a new line at the end of the file. Please enable it in your editor. 

Add this setting if you are using VSCode and then save all the files again.

```json
"files.insertFinalNewline": true
``